src/Objs/_metadataEditingConfig.js

```diff
@@ -17,6 +19,10 @@ export const metadataInitialContent = {
   robotsIndex: "yes",
 };
 
+const LoadableSocialCardsTab = loadable(() =>
+  import("../Components/EditorTabs/SocialCardsTab")
+);
```

Comment:
I understand the "make it a const first" notation is copied from the docs. However, the docs use the imported components in JSX, so there must be an uppercase variable.
Here, we could make it more compact by inlining. Ideally, we would completely hide the loadable aspect from the reader, by leaving it as an implementation detail to the imported module. But this would probably open the door for accidentally importing the wrong (not loadable) module.

* [ ] FYI consider inlining
* [ ] FYI maybe consider hiding the loadable aspect from configs

Reply:
> But this would probably open the door for accidentally importing the wrong (not loadable) module.

That is also what I was thinking. With the inlined variant it now becomes quite obvious, that this is not directly imported.